whydoge.html

```diff
@@ -0,0 +1,190 @@
+                        Run a Full Node
+                    </h3>
+                    <div class="text">
+                        A full node is a program that validates transactions on the Dogecoin blockchain. Since Dogecoin is fully decentralized,  meaning there is no central authority that owns or operates it, the network has to be supported and maintained entirely by individual computers running full nodes. <a class="btn btn-sm btn-secondary" href="https://www.reddit.com/r/dogecoin/comments/l7q640/why_running_a_full_node_is_the_secondbest/" target="_blank">Learn more here</a> about running a full node and how you can do your part to keep the Dogecoin network barking. 
+                    </div>
+                </div>
+            </div>
+
+            <div class="col-lg-4 col-md-6 mb-4">
+                <div class="support-content">
+                    <i class="fa fa-globe text-secondary mb-2"></i>
+                    <h3 data-i18n="dgc-support-contribute-to-website" class="heading mb-4">
+                        Contribute to dogecoin.com
+                    </h3>
+                    <div class="text">
+                        Dogecoin.com is free software. If you are a web developer and a devoted shiba, you can help maintain the official dogecoin website and add features. 
+                    </div>
+                    <a class="btn btn-sm btn-secondary mt-1" href="https://github.com/dogecoin/dogecoin.com" target="_blank">Github</a>
+                </div>
+            </div>
+
+            <div class="col-lg-4 col-md-6 mb-4">
+                <div class="support-content">
+                    <i class="fa fa-book text-secondary mb-2"></i>
+                    <h3 data-i18n="dgc-support-educate" class="heading mb-4">
+                        Educate People
+                    </h3>
+                    <div class="text">
+                        Some shibas want to know more than the basics. They need teachers. If you are well-informed about Dogecoin, you can post educational videos about it online, write blog posts, share info on social media, and so much more. Anything that makes people better educated about Dogecoin is good for Dogecoin. <a class="btn btn-sm btn-secondary mt-1" href="https://www.reddit.com/r/dogeducation/" target="_blank">Start Here</a> 
+                    </div>
+                </div>
+            </div>
+
+            <div class="col-lg-4 col-md-6 mb-4">
+                <div class="support-content">
+                    <i class="fa fa-code text-secondary mb-2"></i>
+                    <h3 data-i18n="dgc-support-develop" class="heading mb-4">
+                        Develop
+                    </h3>
+                    <div class="text">
+                        Dogecoin is free software. If you are a software developer, you can use your shibapowers to do good and improve Dogecoin. You can even build amazing new services or software that use Dogecoin.
```

Comment:
You should avoid using words like Shibas and Shibapowers which don't exist in English , I can understand what you meant but the newcomers might not , or maybe single/double quote them as an alternative